ci/snap/ci-deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+sudo apt-get update
+sudo apt-get install -y snapd
+sudo snap install review-tools
+sudo snap install snapcraft --classic
+sudo lxd init --minimal
+# `--use-lxd` applied to use a LXD container instead of a VM, to work around lack of support for KVM on CircleCI VMs.
+snapcraft --use-lxd
+
+#if [[ ${1-} == "--dry-run" || ${CIRCLE_BRANCH-} != "master" ]]; then
+if [[ ${1-} == "--dry-run" ]]; then
+  echo "DEVELOP"
+  ! command -v git-machete
+  sudo snap install git-machete*.snap --dangerous --classic
+  git machete version
+  sudo snap remove git-machete
+else
+  echo "MASTER"
+  echo "$SNAPCRAFT_LOGIN_CREDENTIALS_CONTENTS_BASE64" base64 -d > ~/.snapcraft.login
```

Comment:
Hmmm does it work? 😯 looks like it should be
```suggestion
  echo "$SNAPCRAFT_LOGIN_CREDENTIALS_CONTENTS_BASE64" | base64 -d > ~/.snapcraft.login
```

Reply:
@PawelLipski Yes, i am pretty sure it works because if it didn't it should throw errors while trying to log in.

Reply:
😯

Reply:
Even without the pipe `|`? in the current shape, `base64 -d` are just loose params of `echo`, appended verbatim to `.snapcraft.login` 🤔

Reply:
...which means that `$SNAPCRAFT_LOGIN_CREDENTIALS_CONTENTS_BASE64` are NOT decoded from base64...